taskfiles/k8s.yml

```diff
@@ -34,6 +34,14 @@ tasks:
       # controller-gen won't output to multiple directories and we need a copy
       # of resources for tests, so cp it is.
       - cp ./config/rbac/v2-manager-role/*.yaml ./internal/controller/redpanda/
+      - |
+        controller-gen \
+          paths='./internal/decommissioning/...' \
+          rbac:roleName=decommissioner-role \
+          output:rbac:artifacts:config=config/rbac/decommissioner-role
+      # controller-gen won't output to multiple directories and we need a copy
+      # of resources for tests, so cp it is.
+      - cp ./config/rbac/decommissioner-role/*.yaml ./internal/decommissioning/
```

Comment:
Can the go embed work in relative paths like `../../config/rbac/decommissioner-role`?